ChattoApp/ChattoApp/Source/Text Messages/DemoTextMessageViewModel.swift

```diff
@@ -44,6 +44,9 @@ public class DemoTextMessageViewModelBuilder: ViewModelBuilderProtocol {
     public func createViewModel(textMessage: DemoTextMessageModel) -> DemoTextMessageViewModel {
         let messageViewModel = self.messageViewModelBuilder.createMessageViewModel(textMessage)
         let textMessageViewModel = DemoTextMessageViewModel(textMessage: textMessage, messageViewModel: messageViewModel)
+        // Best place to decide whether user having avart might be in the presenter. Here just for demo purpose
+        // Because we might only want to display avatar when showTail in the nessage bubble
+        textMessageViewModel.avatarImage = UIImage(named: "userAvatar")
```

Comment:
I am not sure how adding `canShowAvatar` to the `ChatItemDecorationAttributes` can make things easier yet. I need to look into the code to see more use cases of `ChatItemDecorationAttributes`. So if it makes sense, I will add this to the decorator attributes later. :) 

I think user can also do this by themselves by subclassing `ChatItemDecorationAttributes`?

However for now, I will suggest to do this in the presenter based on the actual logic of view model and business requirements. User need to create an custom presenter.